github/data_source_github_actions_secrets.go

```diff
@@ -0,0 +1,106 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-github/v29/github"
+	"github.com/hashicorp/terraform/helper/schema"
+	"log"
+)
+
+func dataSourceGithubActionsSecrets() *schema.Resource {
```

Comment:
I initially added this as I thought it might be useful in future resources / data sources for GitHub actions. Reviewing the available API endpoints I think I agree that this data is useless in the long run. I will remove this data source.